site/blogs/blitsPreCompilation.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: blog
+title: Pre-compilation added to Blits
+description: Blits v0.6.0 has been released and introduces a new pre-compilation feature to improve the performance
+date: 2024-1-4
+---
+
+2024 is starting with a fresh new Blits feature: **pre-compilation** for improved performance.
+
+This feature was already announced at the RDK summit when Lightning 3.0 was presented, and now it has landed in the framework.
+
+Blits comes with a nice and readable XML-based template structure. This template and all the attached data binding for reactivity, are then being parsed and compiled into vanilla javascript instructions that the Lightning 3 renderer can understand.
+
+Until now, this compilation step has been happening _runtime_, on the device. And although we haven’t really seen any noticeable lags on the RPi3 (our reference device), it’s evident that any additional on-device processing can negatively impact performance. Particularly on lower-end devices, this can become an issue.
```

Comment:
maybe add in an example of what that vanilla code looks like? or how its used in Blits? or would that make it too technical?

Reference to Lightning 2.0 where this was a major bottleneck when spawning large screens with a lot of components?